conslogging/conslogging.go

```diff
@@ -151,7 +154,7 @@ func (cl ConsoleLogger) Warnf(format string, args ...interface{}) {
 
 	for _, line := range strings.Split(text, "\n") {
 		cl.printPrefix()
-		c.Fprintf(cl.w, "%s\n", line)
+		c.Fprintf(cl.errW, "%s\n", line)
```

Comment:
`printPrefix` should take an argument for the writer to use so that warnings with a prefix get both the warning and the prefix written to stderr.

Reply:
good catch, I just pushed up a fix for this.